.rubocop.yml

```diff
@@ -6,7 +6,7 @@ require:
   - rubocop-md
 
 AllCops:
-  TargetRubyVersion: 3.1
+  TargetRubyVersion: 3.2
```

Comment:
Could we remove this line and let [rubocop infer it](https://docs.rubocop.org/rubocop/configuration.html#finding-target-ruby-in-a-gemspec-file) from `rails.gemspec`?

Reply:
Thanks for the review. I did not know this feature. 

This feature has been implemtened since RuboCop 1.3.0 in 2020.
So I think this feature is stable and also works at the rails/rails branch.


https://github.com/rubocop/rubocop/blob/v1.3.0/CHANGELOG.md#new-features
https://github.com/rubocop/rubocop/issues/8761